src/Symfony/Component/Validator/Tests/Constraints/ExpressionValidatorTest.php

```diff
@@ -304,4 +307,58 @@ public function testViolationOnPass()
             ->setCode(Expression::EXPRESSION_FAILED_ERROR)
             ->assertRaised();
     }
+
+    public function testIsValidExpression(): void
+    {
+        $constraints = [new NotNull(), new Range(['min' => 2])];
+
+        $constraint = new Expression(
+            ['expression' => 'is_valid(this.data, a)', 'values' => ['a' => $constraints]]
+        );
+
+        $object = new Entity();
+        $object->data = 7;
+
+        $this->setObject($object);
+
+        $this->expectValidateValue(0, $object->data, $constraints);
+
+        $this->validator->validate($object, $constraint);
+
+        $this->assertNoViolation();
+    }
+
+    public function testIsValidExpressionInvalid(): void
+    {
+        $constraints = [new Range(['min' => 2, 'max' => 5])];
+
+        $constraint = new Expression(
+            ['expression' => 'is_valid(this.data, a)', 'values' => ['a' => $constraints]]
+        );
+
+        $object = new Entity();
+        $object->data = 7;
+
+        $this->setObject($object);
+
+        $this->expectFailingValueValidation(
+            0,
+            7,
+            $constraints,
+            null,
+            new ConstraintViolation(
+                'error_range',
+                null, [],
+                null,
+                '',
+                null,
+                null,
+                'range'
```

Comment:
let's put everything on one line and use named arguments to skip all needless defaults?

Reply:
put on one line, named arguments allows to skip only one arg, so leave as is